.github/workflows/build.yaml

```diff
@@ -0,0 +1,16 @@
+name: '🏗️ Build jellyfin-docs'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    uses: Shadowghost/jellyfin-docs/.github/workflows/_meta.yaml@ci-rework
```

Comment:
Shouldn't this (and publish.yaml) point to `jellyfin/jellyfin-docs` instead of `Shadowghost/jellyfin-docs`?

Reply:
Yes but to test it I need to reference an existing workflow :)